src/misc/setup.cpp

```diff
@@ -444,18 +444,17 @@ bool Prop_path::SetValue(const std::string& input)
 		return false;
 	}
 
-	const std::string workcopy = resolve_home(input).string();
+	const std_fs::path workcopy = resolve_home(input);
```

Comment:
Uh, can we use a better name for this? `workcopy` — that can be literally anything... Like `data`, `temp`, `stuff` etc. 😅